.github/workflows/submitBeta.yml

```diff
@@ -0,0 +1,127 @@
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest.json
+          field: name
+          value: Freighter BETA
+      - name: Update manifest.json version_name
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest.json
+          field: version_name
+          value: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+      - name: Update manifest.json version
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest.json
+          field: version
+          value: ${{ github.event.inputs.release }}
+      - name: Build extension
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+      - run: yarn install && yarn build:freighter-api && yarn build:extension:production --define 'process.env.AMPLITUDE_KEY="${{ secrets.AMPLITUDE_KEY }}"' --define 'process.env.SENTRY_KEY="${{ secrets.SENTRY_KEY }}"'
+      - name: Install zip
+        uses: montudor/action-zip@c25e01d7489d0274569440a2f0281b4569df16bc #v0.1.1
+      - name: Zip extension build
+        run: zip -qq -r ./build.zip *
+        working-directory: ./extension/build
+      - name: Commit files
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add -A
+          git commit -m "docs(): bumping release to ${{ github.event.inputs.version }}"
+          git tag ${{ github.event.inputs.version }}
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@7380612b49221684fefa025244f2ef4008ae50ad #v3.10.1
+        with:
+          title: Bump versions to ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+      - name: Create GitHub Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          tag_name: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+          release_name: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+          body: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+          draft: false
+          prerelease: true
+      - name: Upload Release Asset
+        id: upload-release-asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ./extension/build/build.zip
+          asset_name: build-${{ github.event.inputs.version }}.zip
+          asset_content_type: application/zip
+      - name: Build extension for Firefox
```

Comment:
❓  I noticed the submitProduction.yml doesn't have this `Build extension for Firefox` job, but submitBeta does, even though looks like both are submitting to firefox, is that on purpose?

Reply:
Ah good questions: I'm realizing now that I should have annotated this PR to explain what's going on!

So, the idea here is that we're going to build an `.xpi` file, submit to Firefox sign off on it, and then host the build artifact ourselves. 

Self-hosting this .xpi file is Mozilla's solution for beta testing as anyone can install this file by drag/dropping it onto their local Firefox. It's *slightly* easier than unzipping a zip file and loading a temporary add on, thought not as good as Chrome's ability to have a private listing

Reply:
In submitProduction.yml, we're building one zip file and sending them off to Firefox and Chrome to generate whatever file types they want and host it. Here, we actually need to build the xpi file ourself since we're self-hosting. When Firefox is hosting, they do this step behind the scenes